csfieldguide/chapters/content/en/human-computer-interaction/sections/user-experience.md

```diff
@@ -0,0 +1,559 @@
+
+{panel type="teacher-note"}
+
+# Affinity Diagram Tips
+
+This is a very small affinity diagram.
+Your students should have a lot of data to work with from their surveys and therefore should have at least 20 sticky notes that they can sort into 4 or 5 strong themes.
+
+{panel end}
+
+### Step 4: Describe your user.
+
+**UX Methods: Empathy Map, User Persona and User Stories**
+
+Now that we’ve established who our users are and what they want, it would be useful to create a *User Persona*.
+A *User Persona* is a character that is representative of your target user.
+To help us with creating a user persona, we will use an *Empathy Map* to give us a structure to work with.
+An *Empathy Map* defines four aspects of the persona: how they Think and Feel and what they Do and Say, and is drawn in quadrants.
+We’ve created a very basic one below to help us develop a persona for the target user of our sleep habits app:
+
+|                                                                                                                                          	| **User**                                                                                                                                                                                                                    	|
+|------------------------------------------------------------------------------------------------------------------------------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| **Says**<br>Things the user says out loud in an interview or wrote in their survey.<br><br>*I am so tired.<br>I think I stayed up too late last night.* 	| **Thinks**<br>What do they think about? What are their needs, goals and motivations?<br><br>*I wish I could get more sleep.<br>Maybe I should go to bed earlier.<br>How many hours of sleep do I actually need?<br>Why am I always tired?* 	|
+| **Does**<br>What actions does the user do?<br><br>*Always hits “snooze” on the alarm.<br>Drinks coffee first thing in the morning.*                     	| **Feels**<br>Consider the tone they use in an interview and/or survey answers.<br><br>*Tired.<br>Annoyed.<br>Frustrated.*                                                                                                                	|
+
+Using our Empathy Map we are able to derive the target users needs and from here we created a persona representative of our target user.
+Meet Bevan:
+
+**Bevan, 22, Accounting Student at University**
+
+{image file-path="img/chapters/bevan.jpg" alt="Photo of Bevan for persona."}
+
+*__About__*
+
+*Bevan is a busy undergraduate student.
+He gets good grades and plays in a social soccer league.
```

Comment:
Pretty sure you meant the [opposite](https://brilliantmaps.com/football-vs-soccer/) but probably easiest to change it to a different sport, like 'baseball'. Or we could use 'football' because it is a sport in every country (we don't really care what sport it is in this situation), while some people might be confused by the world soccer (in other English speaking countries).

Reply:
That map is 100% incorrect for NZ. But nvm, I meant that while it is football in most countries, football is something completely different in several others, while everyone at least understands that soccer refers to that sport